docs/source/quick_start.rst

```diff
@@ -6,211 +6,117 @@
  Installation
 **************
 
-To install TensorBaySDK using ``PyPI``, run
+To install TensorBay SDK and CLI by ``pip``, run the following command:
 
-.. code:: shell
+.. code:: console
 
-   pip3 install graviti
+   pip3 install tensorbay
 
-*********************************
- Registration and Identification
-*********************************
 
-All TensorBay cloud services require an account. For example, an AccessKey_ is needed for
-identification when using :ref:`quick_start:Read Dataset from TensorBay`. Without registration,
-users can only receive local services, such as :ref:`quick_start:Read Dataset from Local`.
-
-Please visit `Graviti AI Service(GAS)`_ to sign up.
-
-.. _accesskey: https://gas.graviti.cn/access-key
-
-.. _graviti ai service(gas): https://www.graviti.cn/tensorBay
-
-************
- GAS Client
-************
-
-First step to use TensorBay is to create a :ref:`features/tensorbay_client:GAS Client`.
-
-.. code:: python
-
-   >>> from graviti import GAS
-
-   >>> ACCESS_KEY = "Accesskey-*****"
-   >>> gas = GAS(ACCESS_KEY)
-
-*****************************
- Two Methods Using TensorBay
-*****************************
-
-TensorBay provides two methods to connect datasets with your model(see figure below):
-
--  :ref:`quick_start:Read Dataset from TensorBay`.
--  :ref:`quick_start:Read Dataset from Local`.
-
-.. image:: images/read_pipeline.png
-
-Read Dataset from TensorBay
-===========================
-
-In the first method, you can read a dataset uploaded by the community, your workmates, or yourself.
-
-Before reading a dataset from TensorBay, two preparation steps should be done: obtain_ and fork_.
-After that, you only need to pass the correct dataset name to the GAS client, and you will get a
-:ref:`features/tensorbay_client:Dataset Client`.
-
-Here, we take `Dogs vs. Cats`_ as an example.
-
-.. _dogs vs. cats: https://www.graviti.cn/open-datasets/DogsVsCats
-
-.. _fork: https://docs.graviti.cn/guide/opendataset/fork
-
-.. _obtain: https://docs.graviti.cn/guide/opendataset/get
+**************
+ Verification
+**************
 
-.. code:: python
+To verify the SDK and CLI version, run the following command:
 
-   >>> dataset_client = gas.get_dataset( "Dogs vs. Cats")
+.. code:: console
 
-If you are not sure about the dataset name, you can visit our `Opendataset Platform`_ to check all
-"forkable" open datasets. The list method can print all your forked open datasets.
+   gas --version
 
-.. _opendataset platform: https://www.graviti.cn/open-datasets
 
-.. code:: python
+**************
+ Registration 
+**************
 
-   >>> gas.list_dataset_names()
-   ['Dogs vs. Cats', 'nuScenes']
+Users can use local features without registration,
+such as :ref:`quick_start:Create Dataset from Local`.
```

Comment:
:ref:`quick_start:Create Dataset from Local` here is not a feature.

Maybe like this?
```
Users can use local features without registration,
to do tasks such as :ref:`quick_start:Create Dataset from Local`.
```